paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -95,9 +98,20 @@ internal class PaymentSheetPlaygroundViewModel(
                         checkoutResponse.publishableKey
                     )
 
-                    state.value = checkoutResponse.asPlaygroundState(
-                        snapshot = playgroundSettingsSnapshot,
+                    val customerId = checkoutResponse.customerId
+                    val updatedSettings = playgroundSettingsSnapshot.playgroundSettings()
+                    if (
+                        playgroundSettingsSnapshot[CustomerSettingsDefinition] == CustomerType.NEW &&
+                        customerId != null
+                    ) {
+                        println("Customer $customerId")
```

Comment:
I don't think this is ever printed to logs right?

Reply:
I wasn't able to find the intent id, or the customer id anywhere when I was debugging this. But I did find those two values useful, which is why I'm logging them now.